Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManager.cs

```diff
@@ -95,9 +96,23 @@ public override bool CheckCapability(MixedRealityCapability capability)
         protected override GenericXRSDKController GetOrAddController(InputDevice inputDevice)
         {
             GenericXRSDKController controller = base.GetOrAddController(inputDevice);
-            if (controller is OculusXRSDKTouchController oculusTouchController)
+
+            if (!cameraRig.IsNull() && controller is OculusXRSDKTouchController oculusTouchController && oculusTouchController.OculusControllerVisualization == null)
             {
-                oculusTouchController.UseMRTKControllerVisualization = cameraRig.IsNull();
+                GameObject platformVisualization = null; 
+                if (oculusTouchController.ControllerHandedness == Handedness.Left)
+                {
+                    platformVisualization = leftControllerHelper.gameObject;
```

Comment:
If we're now snatching this GameObject, reparenting it, and handling moving it in the scene and whatnot, do we want to go one step further and let MRTK spawn the model in the first place?

Reply:
Asking this more conceptually. Happy to keep the current behavior for this iteration (and approved the PR!)

Reply:
Looking at the code, I'd actually rather have not reparented the entire gameobject, but the TryAddControllerModelToSceneHierarchy seemed to be the right place to set the Visualizer property 😅.

I think since we aren't moving the OVR hands, it's probably better to leave these GameObjects parented to the OVR camera rig for now, and I'll set the visualizer properties manually

`Visualizer = controllerObject.GetComponent<IMixedRealityControllerVisualizer>();

                if (Visualizer != null)
                {
                    Visualizer.Controller = this;
                    return true;
                }
                else
                {
                    Debug.LogError($"{controllerObject.name} is missing a IMixedRealityControllerVisualizer component!");
                    return false;
                }`

Reply:
> it's probably better to leave these GameObjects parented to the OVR camera rig for now

In this case, is there going to be fighting between the MRTK's visualizer script trying to move the controller models with however Oculus's integration handles moving the controller models?

Reply:
Yeah looking at it looks like we might end up setting it's position twice in one call or something like that.

Will parent it for now but also will avoid mucking up our prefabs further for the time being and leave it in the old implementation for this PR.